lib/nconf-truth.ts

```diff
@@ -3,10 +3,10 @@
  */
 
 import * as nconf from 'nconf';
-var common = require('nconf/lib/nconf/common');
+const common = require('nconf/lib/nconf/common');
```

Comment:
can also be TS-ed via either:
* `import common = require(''nconf/lib/nconf/common')`;
* `import * as common ...`